### PR TITLE
Add deno:/status.md

### PR DESCRIPTION
**Pull request by heavenshell**
Show custom status for deno lsp.

https://github.com/denoland/deno/tree/main/cli/lsp#custom-requests

VSCode's behavior

https://user-images.githubusercontent.com/56591/108600079-3e487c00-73d8-11eb-8206-1b78cc082f47.mp4

vim-lsp behavior

https://user-images.githubusercontent.com/56591/108600071-3688d780-73d8-11eb-9ebb-f68eb1c6e9d0.mp4

### CODE REVIEW

**Comment by mattn** [comment]
Looks very useful

File: settings/deno.vim [hunk 1]
```diff
@@ -21,3 +21,57 @@ augroup vim_lsp_settings_deno
...
+    setlocal noswapfile buftype=nofile
+    setlocal nomodified
+    setlocal nomodifiable
+endfunction
+
+function! s:status() abort
+  let l:has_extension = 1
+  let l:command_id = lsp#_new_command()
+  let l:ctx = {
+        \ 'counter': l:has_extension,
```
Comment (mattn) [inline settings/deno.vim:56]: What does this `l:has_extension` mean?

Comment (heavenshell) [reply settings/deno.vim]: oops, sorry.
My experimental code(I'll send Goto defenition PR later) were mixed.
Replace to `l:ctxt = {}`

**Comment by mattn** [comment]
Thank you

**Pull request merged**